docs/appendices/release-notes/unreleased.rst

```diff
@@ -57,6 +57,13 @@ None
 Fixes
 =====
 
+- Fixed a bug that would lead to insertion of records via
+  ``INSERT INTO ... (SELECT ... FROM ..)`` and
+  ``INSERT INTO ... VALUES (...)``  into different partitions while using
+  the same partition by value.
+  This occurs only when the partition key is an object field of the
```

Comment:
can you add an empty line above this to start the new para. also wrap to 79 chars by breaking after `timestamp` if it fits